litex/build/lattice/radiant.py

```diff
@@ -153,7 +153,9 @@ def tcl_path(path): return path.replace("\\", "/")
 
         # Add include paths
         vincpath = ";".join(map(lambda x: tcl_path(x), self.platform.verilog_include_paths))
-        tcl.append("prj_set_impl_opt {include path} {\"" + vincpath + "\"}")
+        if vincpath != "": # Do not add quotes if vincpath is empty, else Radiant complains
+            vincpath = "\"" + vincpath + "\"" 
+        tcl.append("prj_set_impl_opt {include path} {" + vincpath + "}")
```

Comment:
is `include path` is a mandatory option? ie when `vincpath` is empty this line may be skipped?

Reply:
Seems to build fine without `include path`

Reply:
@trabucayre see latest commit. Happy to make further changes / take feedback

Reply:
LGTM, but l.159 must be removed.

Thanks

Reply:
Whoops. Thanks for spotting that.